.travis.yml

```diff
@@ -22,72 +22,140 @@ matrix:
   include:
   - rvm: 2.4.4
     env: PUPPET_GEM_VERSION="~> 5"
+  - rvm: 2.4.4
```

Comment:
puppet6 should be using ruby 2.5.1 - https://puppet.com/docs/puppet/6.0/about_agent.html